lib/mix/tasks/phx.gen.context.ex

```diff
@@ -197,7 +197,51 @@ defmodule Mix.Tasks.Phx.Gen.Context do
 
     paths
     |> Mix.Phoenix.eval_from("priv/templates/phx.gen.context/fixtures.ex", binding)
+    |> Mix.Phoenix.prepend_newline()
     |> inject_eex_before_final_end(test_fixtures_file, binding)
+
+    maybe_print_unimplemented_fixture_functions(context)
+  end
+
+  defp maybe_print_unimplemented_fixture_functions(%Context{} = context) do
+    fixture_functions_needing_implementations =
+      Enum.flat_map(
+        context.schema.fixture_unique_functions,
+        fn
+          {_field, {_function_name, function_def, true}} -> [function_def]
+          {_field, {_function_name, _function_def, false}} -> []
+        end
+      )
+
+    if Enum.any?(fixture_functions_needing_implementations) do
+      Mix.shell.info(
+        """
+
+        Update the following fixture function(s) in #{context.test_fixtures_file}
+        with new implementations:
```

Comment:
```suggestion
        Some of the generated database columns are unique. Please provide
        unique implementations for the following fixture function(s) in
        #{context.test_fixtures_file}:
```

Reply:
I like that much better! I just pushed an update. :slightly_smiling_face: 

Before I merge, I'm also going to spend a little time seeing if I can clean up the actual generator I wrote since I'm thinking it could be a little cleaner.